src/Files.App/Views/ModernShellPage.xaml.cs

```diff
@@ -186,7 +182,8 @@ public ModernShellPage()
             The tool has generated a custom back button in the MainWindow.xaml.cs file.
             Feel free to edit its position, behavior and use the custom back button instead.
             Read: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/case-study-1#restoring-back-button-functionality
-            */ToolbarViewModel.BackRequested += ModernShellPage_BackNavRequested;
+            */
```

Comment:
```suggestion
```

Reply:
Sorry, what exactly should I change?
Should I remove the whole comment or should I change the line as `*/ToolbarViewModel...`

Reply:
@yaichenbaum